lib/gui/lvgl/lv_conf.h

```diff
@@ -257,140 +301,522 @@ typedef void *lv_indev_drv_user_data_t;
 
 /* THEME USAGE */
 
-#ifdef CONFIG_LVGL_THEMES
-
-#define LV_THEME_LIVE_UPDATE	CONFIG_LVGL_THEME_LIVE_UPDATE
+/* Empty theme */
+#ifdef CONFIG_LVGL_THEME_EMPTY
+#define LV_USE_THEME_EMPTY 1
+#else
+#define LV_USE_THEME_EMPTY 0
+#endif
 
-#define LV_USE_THEME_TEMPL 0
+#if LV_USE_THEME_EMPTY
+#define LV_THEME_DEFAULT_FLAG 0
+#define LV_THEME_DEFAULT_INIT lv_theme_empty_init
+#endif
 
-#ifdef CONFIG_LVGL_THEME_DEFAULT
-#define LV_USE_THEME_DEFAULT 1
+/* Material theme */
+#ifdef CONFIG_LVGL_THEME_MATERIAL
+#define LV_USE_THEME_MATERIAL 1
 #else
-#define LV_USE_THEME_DEFAULT 0
+#define LV_USE_THEME_MATERIAL 0
 #endif
 
-#ifdef CONFIG_LVGL_THEME_ALIEN
-#define LV_USE_THEME_ALIEN 1
-#else
-#define LV_USE_THEME_ALIEN 0
+#if LV_USE_THEME_MATERIAL
+#define LV_THEME_DEFAULT_INIT lv_theme_material_init
+
+#if defined(CONFIG_LVGL_THEME_MATERIAL_LIGHT)
+#define LV_THEME_DEFAULT_FLAG LV_THEME_MATERIAL_FLAG_LIGHT
+#elif defined(CONFIG_LVGL_THEME_MATERIAL_DARK)
+#define LV_THEME_DEFAULT_FLAG LV_THEME_MATERIAL_FLAG_DARK
 #endif
 
-#ifdef CONFIG_LVGL_THEME_NIGHT
-#define LV_USE_THEME_NIGHT 1
-#else
-#define LV_USE_THEME_NIGHT 0
 #endif
 
+/* Mono-color theme */
 #ifdef CONFIG_LVGL_THEME_MONO
 #define LV_USE_THEME_MONO 1
 #else
 #define LV_USE_THEME_MONO 0
 #endif
 
-#ifdef CONFIG_LVGL_THEME_MATERIAL
-#define LV_USE_THEME_MATERIAL 1
+#if LV_USE_THEME_MONO
+#define LV_THEME_DEFAULT_INIT lv_theme_mono_init
+#define LV_THEME_DEFAULT_FLAG 0
+#endif
+
+/* Custom theme */
+#if CONFIG_LVGL_THEME_CUSTOM
```

Comment:
This is probably OK but there are situations where Kconfig symbols are undefined, so unless this is guaranteed to get defined it should be:

    #if (CONFIG_LVGL_THEME_CUSTOM - 0)

Reply:
Theme selection is a choice now, so at least 1 theme will be enabled.